nimbus.fml.yaml

```diff
@@ -184,7 +190,8 @@ features:
           },
           "wallpaper-feature": {
             "status": true
-          }
+          },
+          "pocket-sponsored-stories": false
```

Comment:
@lmarceau I set this to false just to try and all the UI tests passed and Unit Tests were not stuck (they failed with the know issue I shared with you yesterday). So, for some reason I don't know yet, enabling break most of the tests and I'm afraid that the app too... not a big thing yet.. but so that we narrow down the problem...

Reply:
So, here's the thing: I don't know what state pocket stories is in, but I know it's not yet a fully operational battleship. There's a PR open to do more work on it from @razvanlitianu , and I don't know if there's more work left to do.

My fix, did change one important thing - `.sponsoredPocket` build level check now returns `true` at the `developer` level, whereas before, it always returned `false`, regardless of level or anything, as it wasn't actually part of Nimbus.

Perhaps by setting this to false, we're just returning to the previous logical flows, which would fix things. Looking at the code, there's one more setting we probably need to update, to test things. I'm gonna try that and push to this branch and see what happens. I bet you it'll fix it. LOL

Reply:
Maybe when that work is done this can be enabled again and the app and tests will be fine... it may be failing because there is something missing...

Reply:
> Walallalalalal!!!! It's green, my friends. Thanks @isabelrios! Your change fixed this. :)
> 
> ![Screen Shot 2022-06-24 at 9 51 49 AM](https://user-images.githubusercontent.com/11182210/175549887-118fd055-a720-4ea7-beb0-7a51c5590a46.png)

yeah, but we are disabling the "pocket-sponsored-stories" :) I hope with the work to be done in @razvanlitianu's PR this can be enabled and everything will be working